Expose RoIAlign to torch

bwasti opened the pull request:
Summary: enable calling roialign (caffe2) from torch frontend

Differential Revision: D13855525

ezyang commented:
One mild suggestion for this set of PRs: we should aspire for our test cases to involve less boilerplate. 50 lines of boilerplate is a lot, for what are very mechanical tests.

bwasti committed:
Enable undefined at::Tensor to be passed as Output

Differential Revision: D13946950

fbshipit-source-id: 604543904e7b12bd0bb57b60f622519c7f4151e6

bwasti committed:
layernorm example

Differential Revision: D13854304

fbshipit-source-id: 2e3cd928a5ee4f80df15b22a9c537c97c80536b6

bwasti committed:
Expose RoIAlign to torch (#16476)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/16476

enable calling roialign (caffe2) from torch frontend

Reviewed By: smessmer

Differential Revision: D13855525

fbshipit-source-id: 71dafc879d1e316c781284d3415229db5cfc324d
